apps/web/templates/application.html.slim

```diff
@@ -6,8 +6,15 @@ html
 
     link rel="stylesheet" href="https://yui-s.yahooapis.com/pure/0.6.0/pure-min.css"
     link href="https://fonts.googleapis.com/css?family=Open+Sans:400,600&amp;subset=cyrillic" rel="stylesheet"
-
+     
+    meta[charset='utf-8']
     meta name="viewport" content="width=device-width, initial-scale=1.0"
+    meta 
+    meta name="twitter:image" content="#{image_url('header-logo.png')}"
+    meta property="og:image" content="#{image_url('header-logo.png')}"
+    meta name="og:url" 
+    meta name="og:description:
```

Comment:
@davydovanton I am sorry, this PR was sent in error, I was saving to grab more details from your site.  Please see questions in the issue.